City-of-Helsinki/drupal-helfi-etusivu
# UHF-10334 article recommendation

# [UHF-10334](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10334)
The implementation missed news article.

## What was done
Added support for news article.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10334`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Log in as admin and enable ai-recommendations block and disable the old one.
- run `drush helfi:annif-entity-defaults`
- run `drush helfi:generate-keywords node news_article`
- run `drush cr`
- Login as admin
- Go edit any article
  - Keywords should have been set
  - show block and show in recommendations checkboxes should be true
- Go view the article
  - You should see recommendations block  




[UHF-10334]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
